app/helpers/embed_player_helper.rb

```diff
@@ -4,18 +4,62 @@ module EmbedPlayerHelper
   EMBED_PLAYER_PATH = "/e"
   EMBED_PLAYER_FEED = "uf"
   EMBED_PLAYER_GUID = "ge"
+  EMBED_PLAYER_CARD = "ca"
+  EMBED_PLAYER_TITLE = "tt"
+  EMBED_PLAYER_SUBTITLE = "ts"
+  EMBED_PLAYER_IMAGE = "ui"
+  EMBED_PLAYER_RSS_URL = "us"
+  EMBED_PLAYER_AUDIO_URL = "ua"
+  DOVETAIL_TOKEN = "_t"
 
-  def embed_player_episode_url(ep, preview = false)
-    if preview && !episode.published?
-      # TODO: private token auth url
+  def embed_player_episode_url(ep, type = nil, preview = false)
+    params = {}
+
+    if preview && !ep.published?
+      params[EMBED_PLAYER_TITLE] = ep.title
+      params[EMBED_PLAYER_SUBTITLE] = ep.podcast.title
+      params[EMBED_PLAYER_IMAGE] = ep.ready_image&.url || ep.podcast.ready_image&.url
+      params[EMBED_PLAYER_RSS_URL] = ep.podcast_feed_url
+      params[EMBED_PLAYER_AUDIO_URL] = enclosure_with_token(ep)
     else
-      embed_params(EMBED_PLAYER_FEED => ep.podcast_feed_url, EMBED_PLAYER_GUID => ep.item_guid)
+      params[EMBED_PLAYER_FEED] = ep.podcast_feed_url
+      params[EMBED_PLAYER_GUID] = ep.item_guid
+    end
+
+    if type == "card" || type == "fixed_card"
+      params[EMBED_PLAYER_CARD] = "1"
     end
+
+    embed_params(params)
+  end
+
+  def embed_player_episode_iframe(ep, type = nil, preview = false)
+    src = embed_player_episode_url(ep, type, preview)
+    allow = "monetization"
+
+    if type == "card"
+      # TODO: this is NOW working, but I'm not sure how helpful this is to a producer that wishes to embed it.
+      tag.iframe src: src, allow: allow, width: "100%", height: "700", style: "--aspect-ratio: 2/3; width: 100%;"
+    elsif type == "fixed_card"
+      tag.iframe src: src, allow: allow, width: "500", height: "700"
+    else
+      tag.iframe src: src, allow: allow, width: "100%", height: "200"
+    end
+  end
+
+  def embed_player_type_options(selected)
+    opts = %w[standard card fixed_card].map { |v| [t("helpers.label.episode.embed_player_types.#{v}"), v] }
+    options_for_select(opts, selected)
   end
 
   private
 
   def embed_params(params)
     "#{play_root}#{EMBED_PLAYER_PATH}?#{params.to_query}"
   end
+
+  def enclosure_with_token(ep)
+    sep = ep.enclosure_url.include?("?") ? "&" : "?"
+    ep.enclosure_url + sep + {DOVETAIL_TOKEN => prx_jwt}.to_query
```

Comment:
Here's the magic that appends a `?_t=<giant-auth-token-string>` to the Dovetail MP3 url.  So you can listen to episodes that are not published yet.